src/algebra/group/units.lean

```diff
@@ -415,6 +415,10 @@ begin
   simp [h.unit_spec]
 end
 
+/-- `is_unit x` is decidable if we can decide if `x` comes from `Mˣ`. -/
+instance {M : Type*} [monoid M] (x : M) : Π [decidable (∃ u : Mˣ, ↑u = x)], decidable (is_unit x) :=
```

Comment:
There is already an `M` variable, no need to redeclare it:
```suggestion
instance [monoid M] (x : M) : Π [decidable (∃ u : Mˣ, ↑u = x)], decidable (is_unit x) :=
```

Reply:
Done.